Formula/pmd.rb

```diff
@@ -11,11 +11,40 @@ class Pmd < Formula
     sha256 "e5ed61e35b5c980b0e1bcf616bc2615f1c5310ad66ce8f4901ca225764e7e78c" => :mavericks
   end
 
+  depends_on :java => "1.8+"
   depends_on "maven" => :build
 
   def install
     ENV.java_cache
 
+    java_version = ENV["JAVA_HOME"][/((\d\.?)+)\.\d/, 1]
+    toolchains = HOMEBREW_CACHE/"java_cache/.m2/toolchains.xml"
+    rm toolchains if File.exist? toolchains
```

Comment:
Except, I guess it needs to be `ENV["_JAVA_OPTIONS"] =` since there was talk of adding `ENV.java_cache` to everything.